src/docs/concepts-development/translations/index.md

```diff
@@ -0,0 +1,191 @@
+  and edited in Flex Console and the API
+published: true
+---
+
+In the FTW templates, user-facing content is not written directly into
+the source code. Instead, the source code uses the
+[ICU message syntax](https://unicode-org.github.io/icu/userguide/format_parse/messages/)
+that defines variables for each meaningful piece of content, and a
+translator or a content creator can define the specific words for the
+variable in their language. The end user only sees the translator's
+words, not the variable itself.
+
+```js
+  // ListingPage.js uses the variable to identify the message
+  const bookingSubTitle = intl.formatMessage({ id: 'ListingPage.bookingSubTitle' });
+...
+  // The wording of the message is defined separately
+  "ListingPage.bookingSubTitle": "Start by choosing your dates.",
+...
+  // This makes it possible to have several translations for a single variable,
+  // so that you can start developing your marketplace in any language, not just English
+  "ListingPage.bookingSubTitle": "Empieza escogiendo las fechas.",
+
+```
+
+Starting from XXXX-XX, marketplace operators can modify the wording of
+the translations in Flex Console. This means that operators can make
+changes to the marketplace texts without the need for code changes. In
+addition, the same translations can now be used from several different
+client applications, making it easier to make centralized changes.
+
+## How translations are handled in Flex
+
+- What is an asset?
+
+With Console-editable translations, Flex introduces a concept of assets.
+Assets are ways to define marketplace content and configurations with
+JSON files without needing to include them in the client application
+codebase.
+
+For the translation version being edited in Flex Console, the asset in
+question is `config/translations.json`. In other words, Flex Console has
+a dedicated view – "Build" > "Content" – for modifying the
+`config/translations.json` asset.
+
+<extrainfo title="Create multiple translation assets">
+If your marketplace has several languages that you would like to modify outside the codebase, you can create specific assets for each language and implement them in your client application in a similar way as the Console editable one. Since only one translation asset can be modified in Flex Console, you will need to use Flex CLI to create and modify the assets.
+</extrainfo>
+
+When the asset has been created, you will need to fetch the translations
+to the client application. If you already have versions of your asset,
+you will use `sdk.assetByVersion`, and if not, you will use
+`sdk.assetByAlias`. TODO: CLARIFY!!
+
+## Translation format for editing translations in Console.
+
+A translation using the
+[React Intl formatMessage formatting](https://formatjs.io/docs/intl#formatmessage) 
+can, at its simplest, consist of a phrase.
+
+```js
+const translations = {
+  en: {
+    "ListingPage.bookingSubTitle": "Start by choosing your dates."
+  }
+}
+...
```

Comment:
![Screenshot 2022-05-11 at 14 27 56](https://user-images.githubusercontent.com/717315/167839017-22d2e3cf-8434-493f-8a13-58a9b95e362a.png)

Related to earlier discussion: I think these examples should be something that they can just copy-paste to Console

Reply:
Fair point, modified